packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/AppContext.kt

```diff
@@ -43,6 +41,10 @@ class AppContext(
     }
   }
 
+  fun onPostCreate() {
+    jsiInterop.installJSI(reactContextHolder.get()!!.javaScriptContextHolder.get())
```

Comment:
When could it fail? Can we catch it?

Reply:
It shouldn't really happen. However, I've replaced it with a little bit safer version.